chore: readme updates & use UDS CLI for zarf

jeff-mccoy opened the pull request:
## Description

...

## Related Issue

Fixes #
<!-- or -->
Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

jeff-mccoy committed:
Merge branch 'main' into jeff-avoiding-keycloak-work

jeff-mccoy committed:
remove zarf & update readme

mjnagel reviewed (dismissed):
https://github.com/defenseunicorns/uds-core/blob/jeff-avoiding-keycloak-work/tasks/deploy.yaml#L21

The bottom half of deploy.yaml also has a number of zarf commands we'll need to update to make CI happy.

jeff-mccoy committed:
more zarf command things

jeff-mccoy committed:
more words

jeff-mccoy committed:
Merge branch 'main' into jeff-avoiding-keycloak-work

jeff-mccoy committed:
Merge branch 'main' into jeff-avoiding-keycloak-work

jeff-mccoy committed:
Update README.md (#143)

Modified some language to highlight the goodness with compliance we ship
(via OSCAL)

Suggested an area where we add a link to our RKE2 prod deployment as an
example of how to move past dev/test and to prod

## Description

...

## Related Issue

Fixes #
<!-- or -->
Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide
Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request)
followed

Co-authored-by: Austen Bryan <102603079+austenbryan@users.noreply.github.com>

jeff-mccoy committed:
Update README.md

mjnagel commented on README.md:
```diff
@@ -1,121 +1,105 @@
-UDS_ADMIN_TLS_KEY=$(cat admin.key)
-UDS_TENANT_TLS_CERT=$(cat tenant.crt)
-UDS_TENANT_TLS_KEY=$(cat tenant.key)
+While the UDS Bundles above can be use for dev and test environments and include a K3d cluster, UDS Core publishes a UDS Package that is inteded to be used with your own UDS Bundle. You can use the [k3d-core-demo bundle](./bundles/k3d-standard/README.md) as an example.
```
```suggestion
While the UDS Bundles above can be use for dev and test environments and include a K3d cluster, UDS Core publishes a UDS Package that is intended to be used with your own UDS Bundle. You can use the [k3d-core-demo bundle](./bundles/k3d-standard/README.md) as an example.
```

mjnagel replied:
Also think we might need to move this below quickstart/dev/test or reword? `While the UDS Bundles above` doesn't flow well with the order as it is.

mjnagel commented on README.md:
```diff
@@ -1,121 +1,105 @@
 
-UDS core publishes one package:
+> [!NOTE]
+> UDS Bundles are intended for dev and test environments and should not be used for production. They also serve as examples to create custom bundles.
```
```suggestion
> These UDS Bundles are intended for dev and test environments and should not be used for production. They also serve as examples to create custom bundles.
```
nit: Just for clarity maybe, since UDS Bundles in general are okay 😄

mjnagel commented on bundles/k3d-standard/README.md:
```diff
@@ -1,25 +1,6 @@
-- [Velero](https://velero.io/) - Backup & Restore
+This bundle is used for demonstration, development, and testing of UDS Core. In addition to the [UDS Core applications](../../README.md#core-applications), it also includes:
 
 The k3d uds-dev-stack provides:
```
```suggestion
This bundle is used for demonstration, development, and testing of UDS Core. In addition to the [UDS Core applications](../../README.md#core-applications), the included k3d uds-dev-stack provides:
```

mjnagel reviewed (commented):
Couple small comments, overall a good improvement to the docs and using uds-cli for zarf. Apologies fo rthe slow review!

mjnagel committed:
docs: wording tweaks

zachariahmiller commented on README.md:
```diff
@@ -1,121 +1,105 @@
+#### tl;dr - [try it now](#quickstart)
 
-The core applications are:
+#### Core applications:
```
If this isnt a sentence any more should it be `Core Applications:` ?

zachariahmiller commented on README.md:
```diff
@@ -1,121 +1,105 @@
-| [Zarf](https://github.com/defenseunicorns/zarf/releases)       | 0.32.1          |
-| [UDS CLI](https://github.com/defenseunicorns/uds-cli/releases) | 0.6.2           |
-| [NodeJS](https://nodejs.org/en/download/)                      | LTS or Current  |
+#### Future applications:
```
Same question as for Core Applications

zachariahmiller commented on README.md:
```diff
@@ -1,121 +1,105 @@
 
-> [!WARNING]  
-> UDS Core is in early alpha and is not ready for general use.
+UDS Core establishes a secure baseline for cloud-native systems and ships with compliance documentation and first-class support for airgap/egress-limited systems. Based on the work of [Platform One](https://p1.dso.mil), UDS Core expands on the security posture of [Big Bang](https://repo1.dso.mil/big-bang/bigbang) while providing advanced automation with the [UDS Operator](./src/pepr/operator/READ ...
```
```
 UDS Core is a collection of several individual applications into a single [Zarf](https://zarf.dev) package and we recommend using [UDS CLI](https://github.com/defenseunicorns/uds-cli?tab=readme-ov-file#install) to deploy it as a [UDS Bundle](#using-uds-core-in-production).
Comment
```

This `several individual applications into a single Zarf package` is in need of a verb

zachariahmiller reviewed (commented):
see comments otherwise looks good.

mjnagel committed:
docs: pr review comments

jeff-mccoy committed:
Update README.md
